test/integration/dedupe.spec.js

```diff
@@ -198,5 +200,22 @@ describe('Request deduping', function() {
             ]);
         }, done, partDone);
     });
+
+    it("deduplicates gets with overlapping ranges", function (done) {
+        var onGet = sinon.spy();
+        var model = new falcor.Model({ source: new LocalDataSource(cacheGenerator(0, 3), { wait: 0, onGet: onGet }) });
+
+        var partDone = after(3, function() {
```

Comment:
Since we already have `lodash` in this package, I'd just as soon use [`_.after`](https://lodash.com/docs/4.17.11#after) so we can cut down on having this extra dep. But I also see the appeal of consistency with the rest of the codebase. It's not a big deal either way.

Reply:
Good catch, the `after` package is actually only used here. I'll remove it in lieu of lodash.